app/models.py

```diff
@@ -1966,6 +1966,10 @@ def create(cls, user_id, name=None, **kwargs):
 
         return super().create(user_id=user_id, name=name, code=code, **kwargs)
 
+    @classmethod
+    def deleteall(cls, user_id):
```

Comment:
I think it'd be more correct grammatically to name the method `delete_all` instead of `deleteall` :)

Reply:
Done